nuxt.config.js

```diff
@@ -35,8 +35,9 @@ export default {
 					})
 
 					callback(null, pageRoutes.concat(postRoutes))
-				}), function (err) {
-					return next(err)
+				}))
+				.catch((error) => {
+					throw new Error(error.response)
```

Comment:
Based on the [Docs](https://nuxtjs.org/api/configuration-generate/#function-with-a-callback) it would seem that you should be passing the `callback` as the argument to `catch`.

Reply:
Should I use `.catch(callback)` as a viable solution for now